terraform/projects/app-ecs-services/alertmanager-service.tf

```diff
@@ -7,9 +7,28 @@
 *
 */
 
+## Variables
+variable "mesh_urls" {
+  type = "list"
+
+  default = ["mesh-1", "mesh-2", "mesh-3"]
+
+}
+
+variable "prometheis_total" {
+  type        = "string"
+  description = "Desired number of prometheus servers.  Maximum 3."
+  default     = "3"
+}
+
 ## Locals
 locals {
   alertmanager_public_fqdns = "${data.terraform_remote_state.app_ecs_albs.alerts_public_record_fqdns}"
+  alertmanager_mesh = "${formatlist("--cluster.peer=%s.${local.private_subdomain}:9094", var.mesh_urls)}"
```

Comment:
Optional: If this is using `local.private_subdomain` which is defined a few lines below it might make sense to move `private_subdomain` up the order so it is defined before being used (at least in sense of reading the code).